src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -16,35 +16,34 @@ internal static class ModuleUtils
         //  - Ignore files/directories when access is denied;
         //  - Search top directory only.
         private static readonly System.IO.EnumerationOptions s_defaultEnumerationOptions =
-                                        new System.IO.EnumerationOptions() { AttributesToSkip = 0 };
+                                        new System.IO.EnumerationOptions() { AttributesToSkip = FileAttributes.Hidden };
```

Comment:
Sorry, I skipped that there is other two `System.IO`
Below too.

Reply:
EnumerationOptions requires the namespace since it collides with the same class in System.Management.  There's one left in the source I can change.